README.md

```diff
@@ -81,6 +81,11 @@ Recreates the `targets` manifest based on the files in `repository/targets`.
 
 Removes all staged manifests and targets.
 
+#### `tuf root-keys`
+
+Outputs a JSON serialized array of root keys to STDOUT. The resulting JSON
+should be distributed to clients for performing initial updates.
```

Comment:
So the Python implementation doesn't check signatures when loading local files, so it just requires a local root.json to be present for the initial update (see [here](https://github.com/theupdateframework/tuf/blob/develop/tuf/client/updater.py#L235-L246)). I was planning a similar thing, just using a non-signed file to start with (e.g. `tuf.NewClient(local, remote).Init([]*data.Key)`) and then using the local root.json in subsequent updates.

I guess your comment is hinting at also verifying local files. This would require shipping root-keys.json and the client then keeping it in sync with any new root.json it downloads (to support revoking root keys for example).

WDYT?